changelog.txt

```diff
@@ -1,5 +1,5 @@
 == Changelog ==
-
+* Fix - Prevent Tribe Commerce 'Confirmation email sender name' from displaying improperly when a single quote is added. [ET-1134]
```

Comment:
The entry should go to `readme.txt` to the appropriate version. Then, once packaging before release, the release engineer (Gustavo) will take care of putting it in the right place. If you want, in any case, you can sync the readme entries here as part of your PR.

Reply:
I just updated my branch and the readme.txt file has the notes in it as well